further_link/util/bluetooth/device.py

```diff
@@ -0,0 +1,166 @@
+import asyncio
+import logging
+from typing import Dict, Optional, Union
+
+from bless import BlessGATTCharacteristic, BlessServer
+
+from further_link.util.bluetooth.gatt import GattConfig
+from further_link.util.bluetooth.messages import ChunkedMessage
+
+
+def to_byte_array(value: str) -> bytearray:
+    return bytearray(str(value), "utf-8")
+
+
+def decode_value(value) -> str:
+    if not isinstance(value, str):
+        value = str(value, "utf-8")
+    return value
+
+
+class BluetoothDevice:
```

Comment:
i think name BluetoothServer would be clearer that there is only ever one of these that handles the different client device connections